src/adapter/src/coord/timestamp_selection.rs

```diff
@@ -95,6 +97,13 @@ impl<S: Append + 'static> Coordinator<S> {
         // If the timestamp is greater or equal to some element in `since` we are
         // assured that the answer will be correct.
         if since.less_equal(&candidate) {
+            event!(
+                Level::TRACE,
```

Comment:
Due to our honeycomb use TRACE can be too much even with just `mz_adapter=trace,info` (honeycomb starts sampling). Maybe better to use DEBUG? I really don't know.

Reply:
I think debug is probably good.